data/mods/ssb/items.ts

```diff
@@ -1,147 +1,71 @@
+
+	// Kalalokki
+	kalalokkiumz: {
+		name: "Kalalokkium Z",
 		isNonstandard: "Custom",
 		onTakeItem: false,
-		zMove: "Great Eqake",
-		zMoveFrom: "Earthquake",
-		itemUser: ["Marowak"],
-		gen: 7,
-		desc: "If held by a Marowak with Earthquake, it can use Great Eqake.",
+		zMove: "Gaelstrom",
+		zMoveFrom: "Blackbird",
+		itemUser: ["Wingull"],
+		gen: 8,
+		desc: "If held by a Wingull with Blackbird, it can use Gaelstrom.",
 	},
-	// SamJo
-	thicciniumz: {
-		name: "Thiccinium Z",
+
+	// Robb576
+	modium6z: {
+		name: "Modium-6 Z",
 		isNonstandard: "Custom",
 		onTakeItem: false,
-		zMove: "Extra T h i c c",
-		zMoveFrom: "Thicc",
-		itemUser: ["Mamoswine"],
-		gen: 7,
-		desc: "If held by a Mamoswine with Thicc, it can use Extra T h i c c.",
+		zMove: "Integer Overflow",
+		zMoveFrom: "Photon Geyser",
+		itemUser: ["Necrozma-Ultra"],
+		gen: 8,
+		desc: "If held by a Robb576 with Photon Geyser, it can use Integer Overflow.",
 	},
-	// Schiavetto
-	mariahcariumz: {
-		name: "Mariahcarium Z",
-		isNonstandard: "Custom",
-		onTakeItem: false,
-		zMove: "Plurshift",
-		zMoveFrom: "Poison Jab",
-		itemUser: ["Scolipede"],
-		gen: 7,
-		desc: "If held by a Scolipede with Poison Jab, it can use Plurshift.",
-	},
-	// Snaquaza
-	fakeclaimiumz: {
-		name: "Fakeclaimium Z",
-		isNonstandard: "Custom",
-		onTakeItem: false,
-		zMove: "Fake Claim",
-		// @ts-ignore hack for Snaquaza's Z move.
-		zMoveFrom: ["Brave Bird", "Superpower", "Sucker Punch", "Flamethrower", "Ice Beam", "Thunderbolt"],
-		itemUser: ["Honchkrow"],
-		gen: 7,
-		desc: "If held by a Honchkrow, it can use Fake Claim.",
-	},
-	// The Immortal
-	buzzniumz: {
-		name: "Buzznium Z",
-		isNonstandard: "Custom",
-		onTakeItem: false,
-		zMove: "Ultra Succ",
-		zMoveFrom: "Drain Punch",
-		itemUser: ["Buzzwole"],
-		gen: 7,
-		desc: "If held by a Buzzwole with Drain Punch, it can use Ultra Succ.",
-	},
-	// Teclis
-	darkrainiumz: {
-		name: "Darkrainium Z",
-		isNonstandard: "Custom",
-		onTakeItem: false,
-		zMove: "Absolute Configuration",
-		zMoveFrom: "Dark Void",
-		itemUser: ["Darkrai"],
-		gen: 7,
-		desc: "If held by a Darkrai with Dark Void, it can use Absolute Configuration.",
-	},
-	// XpRienzo
-	charcoal: {
+
+	// xJoelituh
+	rarebone: {
 		inherit: true,
-		zMove: "Bleh Flame",
-		zMoveFrom: "Blue Flare",
-		itemUser: ["Reshiram"],
-		desc: "Fire-type attacks have 1.2x power. Reshiram with Blue Flare can use Bleh Flame.",
+		onModifyAtkPriority: 2,
+		onModifyAtk() {
+			return this.chainModify(1.5);
+		},
+		onModifyDefPriority: 2,
+		onModifyDef() {
+			return this.chainModify(1.5);
+		},
+		onModifySpDPriority: 2,
+		onModifySpD() {
+			return this.chainModify(1.5);
+		},
+		desc: "1.5x to Attack, Defense, and Special Defense.",
+		// Instruct note: We have a superfluous shortDesc so it is easier to
+		// access the item's description.
+		// But changing desc in mods for items shouldnt be an issue.
+		shortDesc: "1.5x to Attack, Defense, and Special Defense.",
```

Comment:
Something that I want to point out because it somehow was a roadblock when I was coding ssb recently, but modifying the desc property of an existing in the mod folder does not seem to actually change how PS is and still uses the “official” description of the item. You can already see this bug when you do `!dt magnarizer, ssb` on the main PS server when the description for the item was modified in the ssb folder's items.ts (see: https://github.com/smogon/pokemon-showdown/blob/master/data/mods/ssb/items.ts#L23).